appeals/api/src/server/endpoints/documents/documents.controller.js

```diff
@@ -185,9 +185,13 @@ const getStorageInfo = (docs) => {
  */
 const updateDocuments = async (req, res) => {
 	const { body } = req;
-
 	try {
-		await documentRepository.updateDocuments(body.documents);
+		const documents = body.documents;
+		for (const document of documents) {
+			const latestDocument = await documentRepository.getDocumentById(document.id);
+			document.latestVersion = latestDocument?.latestDocumentVersion?.version;
```

Comment:
What's the reason for mapping this value? Couldn't you just reference `latestDocument?.latestDocumentVersion?.version` directly?

Reply:
It is processed as a promise all, so I create an array of objects for the updateAll to process and add the version for each